packages/react-link/package.json

```diff
@@ -33,18 +33,26 @@
     "@types/jest": "~24.9.0",
     "@types/react": "16.9.42",
     "@types/react-dom": "16.9.10",
+    "@types/react-test-renderer": "^16.0.0",
     "@types/webpack-env": "1.16.0",
     "@fluentui/scripts": "^1.0.0",
     "@fluentui/jest-serializer-merge-styles": "^8.0.0-beta.5",
     "enzyme": "~3.10.0",
     "enzyme-adapter-react-16": "^1.15.0",
     "react": "16.8.6",
     "react-app-polyfill": "~1.0.1",
-    "react-dom": "16.8.6"
+    "react-dom": "16.8.6",
+    "react-test-renderer": "^16.3.0"
   },
   "dependencies": {
+    "@fluentui/keyboard-key": "^0.2.13",
     "@fluentui/react-compose": "^1.0.0-beta.12",
+    "@fluentui/react-hooks": "^8.0.0-beta.11",
+    "@fluentui/react-make-styles": "^0.2.5-0",
+    "@fluentui/react-theme-provider": "^1.0.0-beta.22",
+    "@fluentui/react-utils": "^0.3.1-0",
     "@fluentui/set-version": "^8.0.0-beta.1",
+    "@fluentui/theme": "^2.0.0-beta.14",
```

Comment:
Let's avoid a dependency on "@fluentui/theme", as I see a single usage is https://github.com/microsoft/fluentui/pull/16943/files#r574408185 💭

Reply:
I've removed the dependency.